fireplace/actions.py

```diff
@@ -705,9 +707,24 @@ class Discover(TargetedAction):
 	ARGS = ("TARGET", "CARDS")
 
 	def get_target_args(self, source, target):
+		# use hero class for Discover if not neutral (eg. Ragnaros)
+		if target.hero.data.card_class != CardClass.NEUTRAL:
```

Comment:
So noted, I'll try to remember to put comments inside ifs in future (20 years of doing it the other way is going to be rough :P)